commons/tools/warp10.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Access Logs
+shortdesc: Get and process your access logs.
+tags:
+- apps, GTS, warp10, Quantum, visualization
+---
+
+
+# Warp10 
+
+## Main concepts :
+
+Warp10 is a time series database. The notion of `class`, `labels`, `longitude`, `latitude`, `altitude` and `value` are used.
+
+A GTS is a GeoTime Serie define by a `class` and somes `labels`. They are indexed and used to quickly retrived the data.
+
+`labels` is a kind of dictionnary. That is called a **map** under the warp10 terminology
+
+A GTS may contain some values which have the following models : `[ timestamp longitude latitude altitude value ]`
+
+The `warpscript` langage is the langage run by warp10. It is a **reverse polish notation** with the notion of **stack**
+
+> The metrics ovh team build an other way to request warp10: https://github.com/ovh/tsl/
+
+> Notice that the timestamp is in **milliseconds**
+
+> [Warp1O documentation is availlable in their website](https://www.warp10.io/doc/reference)
+
+## Endpoint
+
+The clevercloud warp10 endpoint is:
+
+```
+https://c1-warp10-clevercloud-customers.services.clever-cloud.com/api/v0
+```
+
+You can find some documentation about endpoint gateway [here](https://www.warp10.io/content/03_Documentation/03_Interacting_with_Warp_10/01_Introduction).
```

Comment:
You can find documentation about endpoint gateway [here](https://www.warp10.io/content/03_Documentation/03_Interacting_with_Warp_10/01_Introduction).